cognite/client/data_classes/cdm/__init__.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from cognite.client.data_classes.cdm.v1 import (
+    Activity,
+    ActivityApply,
+    Asset,
+    AssetApply,
+    AssetType,
+    AssetTypeApply,
+    Connection3D,
+    Connection3DApply,
+    Describable,
+    DescribableApply,
+    Equipment,
+    EquipmentApply,
+    Model3D,
+    Model3DApply,
+    Object3D,
+    Object3DApply,
+    Schedulable,
+    SchedulableApply,
+    Sourceable,
+    SourceableApply,
+    TimeSeriesBase,
+    TimesSeriesBaseApply,
+)
+from cognite.client.utils._experimental import FeaturePreviewWarning
+
+__all__ = [
+    "Activity",
+    "ActivityApply",
+    "Asset",
+    "AssetType",
+    "AssetTypeApply",
+    "AssetApply",
+    "Connection3D",
+    "Connection3DApply",
+    "Describable",
+    "DescribableApply",
+    "Equipment",
+    "EquipmentApply",
+    "Model3D",
+    "Model3DApply",
+    "Object3D",
+    "Object3DApply",
+    "Schedulable",
+    "SchedulableApply",
+    "Sourceable",
+    "SourceableApply",
+    "TimeSeriesBase",
+    "TimesSeriesBaseApply",
+]
+
+FeaturePreviewWarning("alpha", "alpha", "Core Data Model").warn()
```

Comment:
Let's remove these imports and move the FeaturePreviewWarning into `cdm/v1.py`. When v2 comes along, it will have many naming collisions with these, so it's best to just keep their namespaces separate.